packages/compass-components/src/components/drawer-portal.tsx

```diff
@@ -175,6 +176,14 @@ const drawerLayoutFixesStyles = css({
     borderTop: 'none',
     borderBottom: 'none',
   },
+
+  // drawer content > title content
+  '& > div:nth-child(2) > div:nth-child(2) > div:first-child > div:first-child > div:first-child > div:first-child':
```

Comment:
🤣

Reply:
Yeah, I'll report this one to leafygreen, and we don't have enough control there to target this otherwise unfortunately (has is still very nice for this, but breaks jsdom)